contracts/src/v0.8/dev/ConfirmedOwnerUpgradeable.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/OwnableInterface.sol";
+import "./vendor/@openzeppelin/contracts-upgradeable/v4.8.1/contracts/proxy/utils/Initializable.sol";
+
+/**
+ * @title The ConfirmedOwner contract
+ * @notice A contract with helpers for basic contract ownership.
+ */
+contract ConfirmedOwnerUpgradeable is Initializable, OwnableInterface {
+  address private s_owner;
+  address private s_pendingOwner;
+
+  event OwnershipTransferRequested(address indexed from, address indexed to);
+  event OwnershipTransferred(address indexed from, address indexed to);
+
+  /**
+   * @dev Initializes the contract in unpaused state.
```

Comment:
Note: same contract as `ConfirmedOwner`, but extends `Initializable` and swaps `constructor` for `initialize` function